sdk/storage/storage-blob/src/Clients.ts

```diff
@@ -4511,12 +4512,12 @@ export class PageBlobClient extends BlobClient {
    *
    * @param {number} offset Starting byte position of the page blob
    * @param {number} count Number of bytes to get ranges diff.
-   * @param {string} prevSnapshot Timestamp of snapshot to retrive the difference.
+   * @param {string} prevSnapshot Timestamp or URL of snapshot to retrive the difference.
```

Comment:
For Python, it was proposed that this feature will only be exposed as an undocumented parameter on getPageRangesDiff. https://github.com/Azure/azure-sdk-for-python/pull/9691/files#diff-cfe8db5b056e62c6775c15a776384beaR2003.  We should probably keep this implementation but undo the documentation change.

Reply:
I didn't see any conversation there. Any idea why they tend to choose that way?

JS SDK have some customers using managed disk account asking for this support.
I was thinking about update the documentation to emphasize that the URL option is only available for managed disk account.

Reply:
"URL is only available for a MD account", make the description clearer if keeping it.

Reply:
Jeremy, would you help to forward the conversation to lin. At same time, are timestamp and url both work for normal page blob and managed disk, or timestamp work for normal page blob and url only work for managed disk?

Reply:
> Are timestamp and url both work for normal page blob and managed disk, or timestamp work for normal page blob and url only work for managed disk?

URL only works for managed disk account.

Reply:
> JS SDK have some customers using managed disk account asking for this support.

What I've heard is they are only internal customers.  If there are external asks then we might need to reconsider.

Reply:
Will go with no document change for now.